packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -5676,17 +5676,38 @@ RED.view = (function() {
                 filteredNodesToImport = nodesToImport.filter(function (n) {
                     if (n.type === "global-config") {
                         importedGlobalConfig = n
+                        // Do not import it if one exists
+                        // The properties of this one will be merged after import
+                        return false
+                    }
+                    return true
+                });
+            } else {
+                filteredNodesToImport = nodesToImport.filter(function (n) {
```

Comment:
Given we are calling `nodesToImport.filter` in both cases of this if/else block, I would personally have written it as a single nodesToImport.filter call, and then have the `if (existingGlobalConfig)` logic inside the filter. But the end result will be the same.

Reply:
It's true that I could have put the logic in a single filter.